src/core/unit.c

```diff
@@ -5267,6 +5289,59 @@ static int unit_export_log_extra_fields(Unit *u, const ExecContext *c) {
         return r;
 }
 
+static int unit_export_log_rate_limit_interval(Unit *u, const ExecContext *c) {
+        char buf[FORMAT_TIMESPAN_MAX];
+        const char *p;
+        int r;
+
+        assert(u);
+        assert(c);
+
+        if (u->exported_log_rate_limit_interval)
+                return 0;
+
+        if (c->log_rate_limit_interval_usec == 0)
+                return 0;
+
+        p = strjoina("/run/systemd/units/log-rate-limit-interval:", u->id);
+        format_timespan(buf, sizeof(buf), c->log_rate_limit_interval_usec, USEC_PER_SEC);
```

Comment:
hmm, given that this is not intended for humans but is just a way for us to pass data between PID 1 and journald, let's just format this as raw number, i.e. not use format_timespan, but simple asprintf() with USEC_FMT for formatting the raw usec value

Reply:
To judge from what I can see, it's the only way to find out what limits are effectively used so I'd leave it as is to simplify debugging a bit.
```sh
-bash-4.4# systemctl show --property LogRateLimitBurst test --no-pager
LogRateLimitBurst=5

-bash-4.4# ls -l /run/systemd/units/ | grep test.service
lrwxrwxrwx. 1 root root 32 Oct 14 23:09 invocation:test.service -> 14d5e5260e1847ef8c9bd7816843997b
lrwxrwxrwx. 1 root root  2 Oct 14 23:09 log-rate-limit-burst:test.service -> 10
lrwxrwxrwx. 1 root root  4 Oct 14 23:09 log-rate-limit-interval:test.service -> 1min
```

Reply:
Sorry. I picked the wrong property above. The following snippet should illustrate what I was trying to say better:
```
-bash-4.4# systemctl show ts est | grep LogRate
LogRateLimitIntervalUSec=1min
LogRateLimitBurst=10

-bash-4.4# ls -l /run/systemd/units/log-rate*
lrwxrwxrwx. 1 root root 1 Oct 14 23:50 /run/systemd/units/log-rate-limit-burst:test.service -> 5
lrwxrwxrwx. 1 root root 3 Oct 14 23:50 /run/systemd/units/log-rate-limit-interval:test.service -> 30s
```
In other words, sometimes dbus properties aren't the most reliable source of information and it's generally useful to be able to debug `systemd` without manually converting usecs to something readable in `gdb` :-)

Reply:
this loses granularity, as you use second granularity for display.

I think it would be much preferable to just use the raw µs value here. it's not particularly hard to grok such values when debugging

Reply:
Quite frankly, the option is called `LogRateLimitIntervalSec` and it would be strange to expect anything more precise here (I know it's possible to pass milliseconds but who needs that anyway). If that is unsatisfactory, we could always `USEC_PER_MSEC` or `1` or even `0` instead of `USEC_PER_SEC`.

What's more confusing is that the dbus property, which is named `LogRateLimitIntervalUSec` (and some more similar dbus properties) when shown by `systemctl show` isn't really the number of usecs but a human-readable string even though the documentation explicitly says that "the properties shown by the command are generally more low-level, normalized versions of the original configuration settings [and] microseconds is the normalized time unit used by the system and service manager".

> it's not particularly hard to grok such values when debugging

I personally find it hard to guess the meaning of numbers without any clues about units of measurement (especially at 3am).

Reply:
Regarding losing granularity, `USEC_PER_SEC` is currently used in `exec_context_done` to dump the value of `LogRateLimitIntervalSec`. It should probably be replaced as well.

Totally unrelated to this PR, but I've just taken a brief look at the functions dumping properties and in almost all of them I noticed `USEC_PER_SEC`. @poettering should that be changed too?